multi/imxrt-multi/spi.c

```diff
@@ -462,8 +462,9 @@ static int spi_muxVal(int spi, int mux)
 		case pctl_mux_gpio_disp_b2_15 :
 			return 9;
 
+		/* For the current MCU, the mux pctl_mux_gpio_emc_b1_00 (value equal of 0) is invalid. This is a temporary workaround */
 		default :
-			return 8;
```

Comment:
Simply replacing the value will cause a few muxes which are ALT8 to not work (e.g. gpio_emc_b2_01). Additional `case` should be added with specific muxes (enums) and `return 8`